docs/Halogen-HTML.md

```diff
@@ -1742,6 +1777,84 @@ wbr_ :: forall p i node. (HTMLRepr node) => [node p i] -> node p i
 This module enumerates some common HTML attributes, and provides additional
 helper functions for working with CSS classes.
 
+#### `ExistsR`
+
+``` purescript
+data ExistsR (f :: # * -> *)
```

Comment:
Would this be useful inside the `purescript-exists` package, or too specific to Halogen's use case?